### DISCUSSION TITLE
An error occurred after the program was packaged with PyInstaller

### DISCUSSION THREAD

**Discussion by Cd297916757** [GPflow/GPflow#1963] [Q&A]
![2391e25a26141309efca53168535daf](https://user-images.githubusercontent.com/55316960/185876877-93d8babb-de46-4267-ae79-dc6a91b45a11.jpg)

**Comment by st--** [comment]
Hi, an error report would be more appropriate as an [issue](https://github.com/GPflow/GPflow/issues). Just from your screenshot it's basically impossible to figure out what you did, and what happened. It looks though like you're using an outdated version of GPflow - there is no `session_manager` module in the current 2.x-branch!
